### DISCUSSION TITLE
不能登录校园网 (E2620: You are already online.)

### DISCUSSION THREAD

**Discussion by habaneraa** [spencerwooo/bitsrun-rs#13] [Q&A] [answered]
![image](https://github.com/spencerwooo/bitsrun-rs/assets/95517280/9ab760a0-33e0-4c06-91e1-a78f132adbec)

这台机器没有登录过，也不能访问外网 IPv4 地址。使用哑终端登录，得到对方的错误消息 “E2620: You are already online.” 不太清楚原因……

**Comment by spencerwooo** [comment]
请联系您实验室该台机器管理员，因为哑终端登记的缘故，会导致机器处于“假性登陆”状态（如登）。

参考：https://github.com/BITNP/bitsrun/issues/45

**Reply by habaneraa** [reply, marked as answer]
感谢您的解答！[该方法](https://github.com/BITNP/bitsrun/issues/45#issuecomment-1868954361) 确实能够解决该问题。

- 出现假登录之后，需要严格按照 “取消登记”，“登录校园网”，“再次登记” 的顺序操作才能恢复联网
- 登记后，取消校园网登录，再过一段时间可能会回到假登录状况，需要重新操作
- 只登记有线网络哑终端，可能不能连接外网

以上内容仅基于我个人的观察，不一定总适用
